salt/srv/salt/base/calamari-crush-location.py

```diff
@@ -20,14 +20,14 @@ def get_last_crush_location(osd_id):
     key = 'daemon-private/osd.%s/v1/calamari/osd_crush_location' % osd_id
     osd_keyring = '/var/lib/ceph/osd/ceph-%s/keyring' % osd_id
     admin_keyring = '/etc/ceph/ceph.client.admin.keyring'
-    commands = (['sudo', 'ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
+    commands = (['ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
                 ['sudo', 'ceph', '--keyring', admin_keyring, 'config-key', 'get', key],
                 )
     for c in commands:
         proc = Popen(c, stdout=PIPE, stderr=PIPE)
         out, err = proc.communicate()
         if proc.returncode != 0:
-            log.error("Error {0} running {1}:'{2}'".format(
+            log.info("Error {0} running {1}:'{2}'".format(
```

Comment:
IMHO the issue wasn't the severity of the message, but the fact that even on success (i.e. one of the commands worked) we were still printing "Error:" lines.

It would make sense to print the output of each command at DEBUG (if at all), and then only at the end print at ERROR if none of the commands succeeded (i.e. if there really is an error)